grails-app/services/au/org/ala/userdetails/EmailService.groovy

```diff
@@ -42,19 +42,31 @@ class EmailService {
           to user.email
           body (view: '/email/activateAccount',
                 plugin:"email-confirmation",
-                model:[link:getServerUrl() + "activateAccount/" + user.id + "/"  + authKey, orgNameLong: grailsApplication.config.skin.orgNameLong ]
+                model:[userName: user.firstName, link: getServerUrl() + "activateAccount/" + user.id + "/"  + authKey, orgNameLong: grailsApplication.config.skin.orgNameLong ]
           )
         }
     }
 
-    def sendAccountActivationSuccess(def user, def activatedAlerts) {
+    def sendAccountActivationSuccess(user, activatedAlerts) {
```

Comment:
If modifying, then I suggest adding (proper) types to the args (and return type while you're at it). Services should be strongly typed IMHO.

Reply:
This is the only method has `def` "type, I was simply making methods signature consistent.

Reply:
NW can leave as is - consistent is good. Including types in services is one of my pet peeves ;-)